lib/pgrel/active_record/store_chain/hstore_chain.rb

```diff
@@ -13,12 +13,7 @@ class HstoreChain < KeyStoreChain
       #   Model.store(:store).value(1, 2).all
       #   #=>[Model(name: 'first', ...), Model(name: 'second')]
       def value(*values)
-        query = String.new
-        values.length.times do |n|
-          query.concat("avals(#{@store_name}) @> ARRAY[?]")
-          query.concat(' OR ') if n < values.length - 1
-        end
-        update_scope(query, *values.map(&:to_s))
+        update_scope("avals(#{@store_name}) && ARRAY[?]", values.map(&:to_s))
```

Comment:
What about adding some tests (especially the ones that should fail before the change)?